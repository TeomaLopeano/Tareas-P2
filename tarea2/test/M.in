copiaCadena
cantidadEnCadena
estaEnCadena 5
imprimirCadena
insertarAlInicio 5 5.0
primeroEnCadena
removerDeCadena 5
insertarAlFinal 5 5.0
infoCadena 5
insertarAlInicio 3 3.0
insertarAlInicio 2 2.0
insertarAlFinal 8 8.0
copiaCadena
removerDeCadena 3
removerDeCadena 5
removerPrimero
cantidadEnCadena
estaEnCadena 5
imprimirCadena
cadenaDeColCadenas 0
cantidadColCadenas 8
estaEnColCadenas 5 5
insertarEnColCadenas 5 5.0 5
cadenaDeColCadenas 5
removerDeColCadenas 5 5
cadenaDeColCadenas 5
insertarEnColCadenas 5 5.0 3
insertarEnColCadenas 5 15.0 3
insertarEnColCadenas 5 25.0 3
insertarEnColCadenas 5 35.0 3
insertarEnColCadenas 5 45.0 3
insertarEnColCadenas 5 55.0 3
removerDeColCadenas 5 3
cadenaDeColCadenas 3
estaDefinidaActual
agregarAIterador 5
reiniciarIterador
actualEnIterador
avanzarIterador
agregarAIterador 6
agregarAIterador 7
agregarAIterador 8
agregarAIterador 9
agregarAIterador 10
reiniciarIterador
actualEnIterador
avanzarIterador
actualEnIterador
reiniciarEstructuras
copiaCadena
cantidadEnCadena
estaEnCadena 5
imprimirCadena
insertarAlInicio 5 5.0
primeroEnCadena
removerDeCadena 5
insertarAlFinal 5 5.0
infoCadena 5
insertarAlInicio 3 3.0
insertarAlInicio 2 2.0
insertarAlFinal 8 8.0
copiaCadena
removerDeCadena 3
removerDeCadena 5
removerPrimero
cantidadEnCadena
estaEnCadena 5
imprimirCadena
cadenaDeColCadenas 0
cantidadColCadenas 8
estaEnColCadenas 5 5
insertarEnColCadenas 5 5.0 5
cadenaDeColCadenas 5
removerDeColCadenas 5 5
cadenaDeColCadenas 5
insertarEnColCadenas 5 5.0 3
insertarEnColCadenas 5 15.0 3
insertarEnColCadenas 5 25.0 3
insertarEnColCadenas 5 35.0 3
insertarEnColCadenas 5 45.0 3
insertarEnColCadenas 5 55.0 3
removerDeColCadenas 5 3
cadenaDeColCadenas 3
estaDefinidaActual
agregarAIterador 5
reiniciarIterador
actualEnIterador
avanzarIterador
agregarAIterador 6
agregarAIterador 7
agregarAIterador 8
agregarAIterador 9
agregarAIterador 10
reiniciarIterador
actualEnIterador
avanzarIterador
actualEnIterador
Fin