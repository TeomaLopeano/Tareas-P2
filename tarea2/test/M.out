1>
2>La cantidad es 0.
3>5 no está en la cadena.
4>
5>Se insertó (5,5.00) al inicio de la cadena.
6>El primero es (5,5.00)
7>Se removió la primera aparición de 5 de la cadena.
8>Se insertó (5,5.00) al final de la cadena.
9>El elemento es (5,5.00)
10>Se insertó (3,3.00) al inicio de la cadena.
11>Se insertó (2,2.00) al inicio de la cadena.
12>Se insertó (8,8.00) al final de la cadena.
13>(2,2.00)(3,3.00)(5,5.00)(8,8.00)
14>Se removió la primera aparición de 3 de la cadena.
15>Se removió la primera aparición de 5 de la cadena.
16>Se removió el primer elemento de la cadena.
17>La cantidad es 1.
18>5 no está en la cadena.
19>(8,8.00)
20>La cadena 0 es :
21>La cantidad en la cadena 8 es 0.
22>5 no está en la cadena 5.
23>Se insertó (5,5.00) en la cadena 5.
24>La cadena 5 es :(5,5.00)
25>Se removió la primera aparición de 5 de la cadena 5.
26>La cadena 5 es :
27>Se insertó (5,5.00) en la cadena 3.
28>Se insertó (5,15.00) en la cadena 3.
29>Se insertó (5,25.00) en la cadena 3.
30>Se insertó (5,35.00) en la cadena 3.
31>Se insertó (5,45.00) en la cadena 3.
32>Se insertó (5,55.00) en la cadena 3.
33>Se removió la primera aparición de 5 de la cadena 3.
34>La cadena 3 es :(5,45.00)(5,35.00)(5,25.00)(5,15.00)(5,5.00)
35>Actual de it NO está definida.
36>Agregando 5 a it.
37>Reiniciando it.
38>El actual de it es 5.
39>Avanzando en it.
40>Agregando 6 a it.
41>Agregando 7 a it.
42>Agregando 8 a it.
43>Agregando 9 a it.
44>Agregando 10 a it.
45>Reiniciando it.
46>El actual de it es 5.
47>Avanzando en it.
48>El actual de it es 6.
49>Estructuras reiniciadas.
50>
51>La cantidad es 0.
52>5 no está en la cadena.
53>
54>Se insertó (5,5.00) al inicio de la cadena.
55>El primero es (5,5.00)
56>Se removió la primera aparición de 5 de la cadena.
57>Se insertó (5,5.00) al final de la cadena.
58>El elemento es (5,5.00)
59>Se insertó (3,3.00) al inicio de la cadena.
60>Se insertó (2,2.00) al inicio de la cadena.
61>Se insertó (8,8.00) al final de la cadena.
62>(2,2.00)(3,3.00)(5,5.00)(8,8.00)
63>Se removió la primera aparición de 3 de la cadena.
64>Se removió la primera aparición de 5 de la cadena.
65>Se removió el primer elemento de la cadena.
66>La cantidad es 1.
67>5 no está en la cadena.
68>(8,8.00)
69>La cadena 0 es :
70>La cantidad en la cadena 8 es 0.
71>5 no está en la cadena 5.
72>Se insertó (5,5.00) en la cadena 5.
73>La cadena 5 es :(5,5.00)
74>Se removió la primera aparición de 5 de la cadena 5.
75>La cadena 5 es :
76>Se insertó (5,5.00) en la cadena 3.
77>Se insertó (5,15.00) en la cadena 3.
78>Se insertó (5,25.00) en la cadena 3.
79>Se insertó (5,35.00) en la cadena 3.
80>Se insertó (5,45.00) en la cadena 3.
81>Se insertó (5,55.00) en la cadena 3.
82>Se removió la primera aparición de 5 de la cadena 3.
83>La cadena 3 es :(5,45.00)(5,35.00)(5,25.00)(5,15.00)(5,5.00)
84>Actual de it NO está definida.
85>Agregando 5 a it.
86>Reiniciando it.
87>El actual de it es 5.
88>Avanzando en it.
89>Agregando 6 a it.
90>Agregando 7 a it.
91>Agregando 8 a it.
92>Agregando 9 a it.
93>Agregando 10 a it.
94>Reiniciando it.
95>El actual de it es 5.
96>Avanzando en it.
97>El actual de it es 6.
98>Fin.
